### PR TITLE
Clean up dependencies

### PR DESCRIPTION
**Pull request by chriswhong**
- Removes some unused packages
- Updates all packages that generate warnings during `npm install`:

Each commit was manually tested based on the concerns of the updated package, but since we don't have a proper test suite we should take great care to manually test this branch before merging.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,51 +5,48 @@
   "main": "index.html",
   "dependencies": {
     "@fortawesome/fontawesome-free": "^6.2.0",
+    "@mapbox/geojson-extent": "^1.0.1",
```
Comment (rreusser) [inline package.json:8]: Interesting. Is there a difference between this and [@turf/bbox](https://www.npmjs.com/package/@turf/bbox)?

File: src/lib/readfile.js [hunk 2]
```diff
@@ -1,10 +1,12 @@
...
+const topojson = require('topojson-client'),
+  toGeoJSON = require('@tmcw/togeojson'),
   gtfs2geojson = require('gtfs2geojson'),
   csv2geojson = require('csv2geojson'),
   osmtogeojson = require('osmtogeojson'),
   polytogeojson = require('polytogeojson'),
-  geojsonNormalize = require('geojson-normalize');
+  geojsonNormalize = require('@mapbox/geojson-normalize');
+
+console.log(gtfs2geojson);
```
Comment (rreusser) [inline src/lib/readfile.js:9]:
```suggestion
```

File: src/lib/readfile.js [hunk 3]
```diff
@@ -165,11 +167,19 @@ function readFile(f, text, callback) {
         }
       }
     );
-  } else if (fileType === 'gtfs') {
+  } else if (fileType === 'gtfs-shapes') {
+    try {
+      return callback(null, gtfs2geojson.lines(text));
+    } catch (e) {
+      console.log(e);
```
Comment (rreusser) [inline src/lib/readfile.js:174]:
```suggestion
```

File: package.json [hunk 4]
```diff
@@ -5,51 +5,48 @@
...
+    "smash": "0.0.15",
     "store": "1.3.14",
-    "togeojson": "0.5.0",
-    "tokml": "0.3.0",
-    "topojson": "1.6.8",
-    "uglify-js": "2.4.1",
-    "wellknown": "0.3.1",
+    "topojson-client": "3.1.0",
+    "topojson-server": "3.0.1",
+    "uglify-js": "3.17.3",
```
[the remark this answers is not in the document]
Comment (chriswhong) [reply package.json:48]: Good to know, I think I'd like to push for a more complete build system that would handle minification, see #771

File: package.json (hunk 1 above)
Comment (chriswhong) [reply package.json:8]: No idea, opened an issue to investigate this #773

**Pull request merged**